docs/about/contributing/modules.md

```diff
@@ -0,0 +1,395 @@
+# Contributing modules
+
+Learn how to create and contribute Terraform modules to the Coder Registry. Modules provide reusable components that extend Coder workspaces with IDEs, development tools, login tools, and other features.
+
+## What are Coder modules
+
+Coder modules are Terraform modules that integrate with Coder workspaces to provide specific functionality. They are published to the Coder Registry at [registry.coder.com](https://registry.coder.com) and can be consumed in any Coder template using standard Terraform module syntax.
+
+Examples of modules include:
+
+- **IDE integrations**: [`code-server`](https://registry.coder.com/modules/coder/code-server), [`jetbrains-gateway`](https://registry.coder.com/modules/coder/jetbrains-gateway), [`cursor`](https://registry.coder.com/modules/coder/cursor)
```

Comment:
Let's split IDE modules into two categories. Desktop IDEs and Web IDEs
`code-server` will go in Web IDES.